mlflow/tracing/fluent.py

```diff
@@ -194,20 +201,14 @@ def start_span(
         mlflow_span.end()
 
 
-def get_traces(n: int = 1) -> List[Trace]:
+def get_trace(request_id: str) -> List[Trace]:
```

Comment:
Thoughts on retaining the ability to pull from the in-memory TTLCache to preserve this functionality?

```python
def get_traces(n: int = 1):
    
    items = list(TRACE_BUFFER.values())
    
    sorted_items = sorted(items, key=lambda x: x['timestamp_ms'], reverse=True)
    
    return sorted_items[:last_n]
```

Reply:
I think we can leave it until requested... given that `mlflow.search_traces(max_results=n)` returns the same output. In-memory version should be a bit faster...but I guess probably the API is not used so frequently.

Reply:
Based on offline conversation - 100% agreed. Let's leave it for now :)